flink-connectors/flink-connector-base/src/main/java/org/apache/flink/connector/base/source/reader/SingleThreadMultiplexSourceReaderBase.java

```diff
@@ -108,4 +130,26 @@ public SingleThreadMultiplexSourceReaderBase(
             SourceReaderContext context) {
         super(splitFetcherManager, recordEmitter, eofRecordEvaluator, config, context);
     }
+
+    /**
+     * This constructor behaves like {@link #SingleThreadMultiplexSourceReaderBase(Supplier,
+     * RecordEmitter, Configuration, SourceReaderContext)}, but accepts a specific {@link
```

Comment:
we can update this java doc with an end `but accepts a specific  {@link RateLimiterStrategy}.`